schema.sql

```diff
@@ -9,10 +9,32 @@ to create the tables within that database
 */
 
 CREATE TABLE IF NOT EXISTS bookings (
-  id SERIAL PRIMARY KEY,
+  booking_id SERIAL PRIMARY KEY,
   booking_date DATE NOT NULL,
   booking_type VARCHAR(50) NOT NULL,
   business_id VARCHAR(150) NOT NULL,
+  business_name TEXT NOT NULL,
   latitude NUMERIC(10, 5) NOT NULL,
-  longitude NUMERIC(10, 5) NOT NULL
-);
\ No newline at end of file
+  longitude NUMERIC(10, 5) NOT NULL,
+  user_id INT NOT NULL
+);
+
+CREATE TABLE IF NOT EXISTS users (
+  user_id SERIAL PRIMARY KEY,
+  user_name VARCHAR (150) NOT NULL,
+  user_password VARCHAR (50) NOT NULL,
+  is_artist BOOLEAN NOT NULL DEFAULT false,
+  bio TEXT,
+  portrait_url TEXT,
+  website TEXT
+);
+
+CREATE TABLE IF NOT EXISTS follows (
+  follow_id SERIAL PRIMARY KEY,
+  fan_id INT NOT NULL,
+  artist_id INT NOT NULL
+);
+
+ALTER TABLE bookings ADD CONSTRAINT fk_user FOREIGN KEY (user_id) REFERENCES users (user_id);
+ALTER TABLE follows ADD CONSTRAINT fk_fan FOREIGN KEY (fan_id) REFERENCES users (user_id);
+ALTER TABLE follows ADD CONSTRAINT fk_artist FOREIGN KEY (artist_id) REFERENCES users (user_id);
```

Comment:
Could you move the foreign keys into the table creation itself? It's easier to read within the table creation itself than to get to the end of the file and reference back up.